tiered-storage/file-system/src/main/java/org/apache/bookkeeper/mledger/offload/filesystem/impl/FileSystemManagedLedgerOffloader.java

```diff
@@ -299,10 +324,15 @@ public void run() {
                         dataWriter.append(key, value);
                     } catch (IOException e) {
                         ledgerReader.fileSystemWriteException = e;
+                        ledgerReader.mxBean.recordWriteToStorageError(managedLedgerName);
                         break;
                     }
                     haveOffloadEntryNumber.incrementAndGet();
+                    ledgerReader.mxBean.recordOffloadBytes(managedLedgerName, entry.getEntryBytes().length);
                 }
+                long writeEntryTimeInNs = System.nanoTime() - start;
+                ledgerReader.mxBean.recordWriteToStorageLatency(managedLedgerName, writeEntryTimeInNs,
+                        TimeUnit.NANOSECONDS);
```

Comment:
Should we record the bytes count of each upload batch? we can use the counter type